blog/2020-09-30-lazyMoreInfo-viewChanges.md

```diff
@@ -0,0 +1,47 @@
+---
+author: Zack Barett
+authorURL: https://github.com/zsarnett
+authorTwitter: zsarnett
+title: "Lovelace: Lazy Loading More Info and Custom View Layouts"
+---
+
+Custom Element developers can now create Custom View Layouts that users can load and use!
+
+In 0.116, we will be changing the way that we create Views in Lovelace. In the past, we had 2 views, a default view, and a panel view. When talking about adding Drag and Drop to Lovelace we decided we could do even better and start allowing custom view types.
+
+Custom Developers will now be able to create a view that receives the following properties:
+
+```ts
+interface LovelaceViewElement {
+  hass?: HomeAssistant;
+  lovelace?: Lovelace;
+  index?: number;
+  cards?: Array<LovelaceCard | HuiErrorCard>;
+  badges?: LovelaceBadge[];
+  setConfig(config: LovelaceViewConfig): void;
+}
+```
+
+The cards and badges will be already created and HASS will be updated on each card so that you do not need to worry about that in your code.
```

Comment:
```suggestion
The cards and badges will be already created and Hass will be updated on each card so that you do not need to worry about updating Hass on the Cards array for every Hass update.
```